app/helpers/template_helpers.py

```diff
@@ -212,8 +212,10 @@ def render_template(template: str, **kwargs: Mapping) -> str:
 
 
 def get_google_tag_manager_context() -> Mapping:
-    cookie = request.cookies.get("ons_cookie_policy")
-    if cookie and "'usage':true" in cookie:
+    if (
+        current_app.config["EQ_GOOGLE_TAG_MANAGER_ID"]
+        and current_app.config["EQ_GOOGLE_TAG_MANAGER_AUTH"]
+    ):
```

Comment:
Could make use of the walrus operator:
```py
    if google_tag_manager_id := current_app.config["EQ_GOOGLE_TAG_MANAGER_ID"] and (
        google_tag_manager_auth := current_app.config["EQ_GOOGLE_TAG_MANAGER_AUTH"]
    ):
        return {
            "google_tag_manager_id": google_tag_manager_id,
            "google_tag_manager_auth": google_tag_manager_auth,
        }
    return {}
```

Reply:
Looks ok to me, but when I update this the ide complains that `google_tag_manager_auth` is might now be referenced before it's assignment

Reply:
If the py version of your IDE is not set to py 3.8 then it will complain as walrus only got introduced in 3.8